### PR TITLE
added nav bar/search bar

### PR DESCRIPTION
**Pull request by daniela-93**
added nav bar and search bar with bulma

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,158 +1,182 @@
...
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./assets/css/style.css">
-  <title>Recipedia</title>
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
-  <title>Recipidia</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (frecklescherokee) [inline index.html]: Why was this removed at the top of the HTML page?

<!DOCTYPE html>	<meta charset="UTF-8">
<html lang="en">
     <head>

File: index.html [hunk 2]
```diff
@@ -1,158 +1,182 @@
...
+<link rel="stylesheet" href="./assets/css/style.css">
+<title>Recipedia</title>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css">
+<title>Recipidia</title>
```
Comment (frecklescherokee) [inline index.html]: remove line 6.  It repeats line 4 except it is misspelled
<title>Recipidia</title>

File: index.html [hunk 3]
```diff
@@ -1,158 +1,182 @@
...
-    </div>
-    <div>
-      <a href="">Shopping List</a>
-    </div>
-  </header>
 
-  <main class="main">
+<!-- ********** HEADER ********** -->
+<!-- ********** NAV BAR ********** -->
+<nav class="navbar" role="navigation" aria-label="main navigation">
```
Comment (frecklescherokee) [inline index.html]: What happened to the <header> tag?  Why was it removed?

**Review by frecklescherokee — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -1,158 +1,182 @@
...
+
+      <a class="navbar-item">
+      Shopping List
+      </a>
+
     
-    <!-- <div>
-      <h1>Recipedia</h1>
+
+
```
Comment (Sha-94) [inline index.html]: remove spacing

**Review by Sha-94 — CHANGES_REQUESTED** [review]

**Review by Sha-94 — DISMISSED** [review]

**Review by Sha-94 — APPROVED** [review]

**Review by frecklescherokee — APPROVED** [review]
